minetest.conf.example

```diff
@@ -726,6 +728,15 @@
 #    type: bool
 # opaque_water = false
 
+#    World-aligned textures may be scales to span several nodes. However,
+#    the server may not send the scale you want, especially if you use
+#    a specially-designed texture pack; with this option, the client tries
+#    to determine the scale automatically basing on the texture size.
+#    See also min_texture_size.
+#    Warning: this option is EXPERIMENTAL!
+#    type: enum values: disable, enable, force
+# autoscale_mode = disable
```

Comment:
Documentation of `world_aligned_mode` missing in this file. Perhaps complete `minetest.conf.example` and run the converter script in builtin?